src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/PermittedSessionLocationDto.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.PermittedSessionLocation
+
+data class PermittedSessionLocationDto(
+
+  @Schema(description = "Permitted Session Location id", example = "123", required = true)
+  val id: Long,
+
+  @Schema(description = "Level one location code", example = "c", required = true)
+  var levelOneCode: String,
+
+  @Schema(description = "Level two location code", example = "1", required = false)
+  var levelTwoCode: String,
```

Comment:
this and levelThreeCode and level4Code needs to be nullable

Reply:
Will change...